### PR TITLE
Add The Imperium to revive request list

### PR DESCRIPTION
**Pull request by Mephiles**
@Sashank999 Can you check I didn't miss anything?

### CODE REVIEW

**Review by Sashank999 — COMMENTED** [review]
I think thats it.

File: extension/manifest.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		]
 	},
 	"permissions": ["storage", "notifications", "background", "https://api.torn.com/"],
-	"optional_permissions": ["https://www.tornstats.com/", "https://yata.yt/", "https://www.nukefamily.org/", "https://tornuhc.eu/"],
+	"optional_permissions": ["https://www.tornstats.com/", "https://yata.yt/", "https://www.nukefamily.org/", "https://tornuhc.eu/", "https://api.mavriware.me/"],
```
Comment (LeoMavri) [inline extension/manifest.json:33]: This should be "https://api.mavriware.me/revive".

Comment (DeKleineKobini) [reply extension/manifest.json:33]: No, the path doesn't matter for the permission.

**Review by DeKleineKobini — APPROVED** [review]
Looks good.

**Pull request merged**